python/cudf/cudf/io/orc.py

```diff
@@ -26,9 +26,9 @@
 from cudf.utils.dtypes import cudf_dtype_from_pa_type, dtype_to_pylibcudf_type
 
 try:
-    import ujson as json  # type: ignore[import-untyped]
+    import ujson as json
```

Comment:
I was thinking if we should just remove this `ujson` import (I think parquet.py or json.py) has this too). I don't think we explicitly test if these readers/writers work with ujson

Reply:
We don't, yeah. I'd also be fine with removing this too. Done in https://github.com/rapidsai/cudf/pull/20560